bench/tpcc.py

```diff
@@ -3,7 +3,7 @@
 import time
 
 sys.path.insert(0, '.')
-from locomotor import redis_server
+from locomotor import *
```

Comment:
Avoid using `import *`. It's fine if you need to import multiple things, but this should be explicit.